hw2_git_workflow_sequences/homework2_task6.py

```diff
@@ -0,0 +1,12 @@
+"""
+    Task6.You have a group of people with non-unique names.
+    Generate a list of non-duplicate names (you cannot use set for this task).
+    If there are 2 johns in the list, you need to take only one of them.
+    "John Dow", "John Dow", "Marta Dow" => "John Dow", "Marta Dow ")
+"""
+non_unique_names = ['Bob', 'Marley', 'Genna', 'Michael', 'Bob', 'Genna']
+unique_names = []
+for name in non_unique_names:
```

Comment:
at this point, we do not learn loops. So the solution was to using some function of dict

Reply:
Solution with dict fromkeys method added